Dropwizard 1.0.0 doesn't support the subdirectory template inheritance

Hey, 

We just updated the DropWizard from 0.9.3 to 1.0.0. But found the subdirectory mustache inheritance will fail. Below is the detailed behaviour.

Note: In the local development environment it is working, but when we deploy it via `java -jar project-SNAPSHOT.jar server integration`, we found it is not working anymore.

The main layout template: `views/layout.mustache`

```
<!DOCTYPE html>
<html lang="en-gb">
<head>
    <meta charset="utf-8">
    <title>Hello DropWizard</title>
</head>
<body>
{{$body}}{{/body}}
</body>
</html>
```

The sub page which inherent the layout but located in the subdirectory: `views/error-pages/not_found.mustache`

```
{{<../layout}}
{{$body}}
        <h1 class="error-page__title">
            Error 404 - page not found
        </h1>
        <div class="error-page__content">
            <p>We couldn't find the page you're looking for.</p>
        </div>
{{/body}}
{{/../layout}}
```

And this is the view we rendered it: `NotFoundView.java`

```
public class NotFoundView extends View {
    public NotFoundView() {
        super("error-pages/not-found.mustache");
    }
}
```

It was working in DropWizard 0.9.3. But we got the error when trying to reach this page when we upgraded it to 1.0.0.

The error message is below:

```
Caused by: javax.ws.rs.WebApplicationException: HTTP 500 Internal Server Error
	at io.dropwizard.views.ViewMessageBodyWriter.writeTo(ViewMessageBodyWriter.java:89)
	at coop.membership.portal.helpers.ViewMessageBodyWriterWithGlobalContext.writeTo(ViewMessageBodyWriterWithGlobalContext.java:63)
	at coop.membership.portal.helpers.ViewMessageBodyWriterWithGlobalContext.writeTo(ViewMessageBodyWriterWithGlobalContext.java:24)
	at org.glassfish.jersey.message.internal.WriterInterceptorExecutor$TerminalWriterInterceptor.invokeWriteTo(WriterInterceptorExecutor.java:265)
...
```

We suspect it's cause  by the magic `{{<../layout}} ...`, cause it's the only difference with other pages. We solved this problem by pulling the `not_found.mustache` out of the `error_pages` subdirectory.

But this should be an issue introduced by the newest 1.0.0 version.

--- a/dropwizard-bom/pom.xml
+++ b/dropwizard-bom/pom.xml
@@ -181,7 +181,7 @@
             <dependency>
                 <groupId>com.github.spullara.mustache.java</groupId>
                 <artifactId>compiler</artifactId>
-                <version>0.9.3</version>
+                <version>0.9.4</version>
                 <exclusions>
                     <exclusion>
                         <groupId>com.google.guava</groupId>

